lib/train/platforms/detect/specifications/os.rb

```diff
@@ -58,10 +62,14 @@ def self.load
               cmd = @backend.run_command('FastCli -p 15 -c "show version | json"')
               if cmd.exit_status == 0 && !cmd.stdout.empty?
                 require 'json'
-                eos_ver = JSON.parse(cmd.stdout)
-                @platform[:release] = eos_ver['version']
-                @platform[:arch] = eos_ver['architecture']
-                true
+                begin
+                  eos_ver = JSON.parse(cmd.stdout)
+                  @platform[:release] = eos_ver['version']
+                  @platform[:arch] = eos_ver['architecture']
+                  true
+                rescue JSON::ParserError => _
```

Comment:
Don't even need the assignment here, can just be `rescue JSON::ParserError`.

Reply:
force of habit 😁 
removed it and updated the PR